steps/hooks.go

```diff
@@ -31,6 +31,21 @@ var _ = gauge.BeforeScenario(func(exInfo *gauge_messages.ExecutionInfo) {
 	store["namespace"] = namespace
 	store["scenario.cleanup"] = cleanup
 	store["targetNamespace"] = config.TargetNamespace
+
+	// Skip pipelines SA check if scenario has @install tag
+	skipSAWait := false
+	for _, tag := range exInfo.CurrentScenario.Tags {
```

Comment:
It's much easier to create a simple if statement
```
if slices.Contains(exInfo.CurrentScenario.Tags, "install")
```

Reply:
Done